Cargo.toml

```
@@ -221,6 +221,7 @@ cs_serde_bytes = "0.12.2"
 derive-quickcheck-arbitrary = "0.1.1"
 fvm3 = { package = "fvm", default-features = false, version = "~3.8", features = ["arb"] }
 fvm_shared3 = { package = "fvm_shared", version = "~3.6", default-features = false, features = ["arb"] }
+glob = "0.3.1"
```

```suggestion
glob = "0.3"
```

So that depedabot won't change this file on every patch version upgrade